src/createContext.ts

```diff
@@ -60,7 +60,7 @@ export const importBuiltins = (context: Context, externalBuiltIns: CustomBuiltIn
 
   if (context.chapter >= 1) {
     defineSymbol(context, 'runtime', misc.runtime)
-    defineSymbol(context, 'display', externalBuiltIns.display)
+    defineSymbol(context, 'display', (v: Value) => externalBuiltIns.display(v, context.externalContext))
     defineSymbol(context, 'error', misc.error_message)
     defineSymbol(context, 'prompt', externalBuiltIns.prompt)
```

Comment:
Prompt should also be curried with `externalBuitIns`, iirc? It's definition is something like `promptString: string => void`

Reply:
`alert` should also be curried as well. It doesn't exist on the node environment, and so will cause a ReferenceError if a student submits a program with alert.

(github doesn't let me review a line that far away from a diff'd line?)